llvm/lib/Target/X86/X86CompressEVEX.cpp

```diff
@@ -220,6 +220,29 @@ static bool performCustomAdjustments(MachineInstr &MI, unsigned NewOpc) {
   return true;
 }
 
+static bool isRedundantNewDataDest(MachineInstr &MI, const X86Subtarget &ST) {
+  // addq %rax, %rbx, %rbx / addq %rbx, %rax, %rbx -> addq %rax, %rbx
```

Comment:
Better to use Intel syntax to match with operant order.

Reply:
Good idea!

Reply:
Done